apps/studio/src/schemas/user.ts

```diff
@@ -0,0 +1,87 @@
+import { RoleType } from "~prisma/generated/generatedEnums"
+import { z } from "zod"
+
+import { offsetPaginationSchema } from "./pagination"
+
+export const createInputSchema = z.object({
```

Comment:
[nit] actually why do we omit the `User` here (ie, why not `createUserInputSchema`)? i thought we'd err on teh side of verbosity but this isn't a huge issue because this is used in the router - just makes it easier next time to know that it's the correct schema (eg: if we have 2 `createInputSchema`)

Reply:
fixed in https://github.com/opengovsg/isomer/pull/1118/commits/146a8c2a92d5f105eebb40842ff061c01d1ec48a